web/src/components/KebabMenu.tsx

```diff
@@ -83,7 +82,7 @@ const ToggleButton = styled.button`
   margin-top: 6px;
 `
 
-const KebabMenu = ({ items, direction, show, setShow, language }: KebabMenuProps): ReactElement | null => {
+const KebabMenu = ({ items, direction, show, setShow, footer }: KebabMenuProps): ReactElement | null => {
```

Comment:
We don't really have a convention for this yet but I'd follow the react one to use uppercase names for react nodes/elements.
```suggestion
const KebabMenu = ({ items, direction, show, setShow, Footer }: KebabMenuProps): ReactElement | null => {
```